receivers/slack/testing.go

```diff
@@ -3,7 +3,6 @@ package slack
 // FullValidConfigForTesting is a string representation of a JSON object that contains all fields supported by the notifier Config. It can be used without secrets.
 const FullValidConfigForTesting = `{
```

Comment:
I'm not sure what to do with these Yuri. Should I create one for incoming webhooks and another for bot tokens?